src/models/BlockTracker.ts

```diff
@@ -0,0 +1,21 @@
+import mongoose, { Schema, Document } from 'mongoose'
+
+const blockTrackerSchema = new Schema({
```

Comment:
I thinking to rename the model. `blockTracker` sounds like a class.
What about `ProcessedBlock`?